label_studio/projects/urls.py

```diff
@@ -41,6 +41,9 @@
     # Project import
     path('<int:pk>/imports/<int:import_pk>/', api.ProjectImportAPI.as_view(), name='project-imports'),
 
+    # Project reimport
+    path('<int:pk>/reimports/<int:reimport_pk>/', api.ProjectReimportAPI.as_view(), name='project-reimports'),
```

Comment:
Probably need test for this new endpoint?

Reply:
Added in LSE repo